File: xssfinder-core/src/main/java/org/xssfinder/xss/XssAttack.java
package org.xssfinder.xss;

public class XssAttack {
    private static final String ATTACK_TEMPLATE =
            "<script type=\"text/javascript\">" +
                "if (typeof(window.xssfinder) === \"undefined\"){" +
                    "window.xssfinder = [];" +
                "}"+
                "window.xssfinder.push('%s');" +
            "</script>";

    private final String identifier;

    public XssAttack(String identifier) {
        this.identifier = identifier;
    }

    public String getIdentifier() {
        return identifier;
    }

    public String getAttackString() {
        return String.format(ATTACK_TEMPLATE, identifier);
    }
}

File: xssfinder-php-executor/src/XssFinder/Xss/XssAttack.php
<?php

namespace XssFinder\Xss;

class XssAttack
{
    const ATTACK_TEMPLATE = <<<'HTML'
<script type="text/javascript">
  if (typeof(window.xssfinder) === "undefined") {
    window.xssfinder = [];
  }
  window.xssfinder.push('%s');
</script>
HTML;

    private $_identifier;
    private $_attackString;

    /**
     * @param string $identifier
     */
    public function __construct($identifier)
    {
        $this->_identifier = $identifier;
        $this->_attackString = sprintf(preg_replace('/\\n/', '', self::ATTACK_TEMPLATE), $this->_identifier);
    }

    public function getIdentifier()
    {
        return $this->_identifier;
    }

    public function getAttackString()
    {
        return $this->_attackString;
    }
}